demos/resourceflavour-taints-and-tolerations/resources/job.yaml

```diff
@@ -25,4 +25,3 @@ spec:
             memory: "200Mi"
             nvidia.com/gpu: 1
       restartPolicy: Never
-
```

Comment:
Should this deletion be in this PR?

Reply:
It's just to tidy up the additional space break. I could revert it if needed